.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+# Description
+
+Fixes #issue.
+
+# Test Plan
+
+
+# Checklist
+
+If these changes modify code paths involving cryptography, the opening of files in VMs, network (via the RPC service) traffic, or fine tuning GUI changes:
+
+- [ ] I have tested these changes in Qubes OR I do not have a Qubes workstation (the reviewer will need to test your changes work in Qubes)
```

Comment:
Single checkbox with two choices could be a bit confusing to interpret, how about:
> If these changes modify code paths involving cryptography, the opening of files in VMs, network (via the RPC service) traffic, or fine tuning of the graphical user interface, Qubes testing is required. Please check as applicable:
> - [ ] I have tested these changes in Qubes
> - [ ] I do not have a Qubes OS workstation